api/conf/settings/elasticsearch.py

```diff
@@ -2,24 +2,28 @@
 
 from aws_requests_auth.aws_auth import AWSRequestsAuth
 from decouple import config
-from elasticsearch import Elasticsearch, RequestsHttpConnection
+from elastic_transport import RequestsHttpNode
+from elasticsearch import Elasticsearch
 from elasticsearch_dsl import connections
 
 from api.constants.media_types import MEDIA_TYPES
 from conf.settings.aws import AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
 
 
-def _elasticsearch_connect():
+def _elasticsearch_connect() -> tuple[Elasticsearch, str]:
     """
     Connect to configured Elasticsearch domain.
 
     :return: An Elasticsearch connection object.
     """
 
+    es_scheme = config("ELASTICSEARCH_SCHEME", default="http://")
     es_url = config("ELASTICSEARCH_URL", default="localhost")
     es_port = config("ELASTICSEARCH_PORT", default=9200, cast=int)
     es_aws_region = config("ELASTICSEARCH_AWS_REGION", default="us-east-1")
 
+    es_endpoint = f"{es_scheme}{es_url}:{es_port}"
```

Comment:
It is no longer possible to easily construct the endpoint from the Elasticsearch client internals. We do this in tests. That was hacky anyway. It's much clearer and cleaner to just save and export it from the place it's centrally built anyway.

Reply:
Not for this PR, but will we need this endpoint for the ingestion server tests in the future? The functions seem so similar, and the return is one of the biggest differences.

Reply:
Maybe? The ingestion server tests already constructs its own ES connection, it doesn't re-use the code's, so I don't know how relevant each implementation is to the other at the moment (not to mention we don't have the facility to share Python code between projects yet, I don't think).